resolve-version/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
```

Comment:
Ah could you also add a Dependabot Cargo entry now there is Rust code in this repo? :-)

Reply:
Good catch. Added in [931b33a](https://github.com/heroku/heroku-buildpack-nodejs/pull/1482/commits/931b33ae28af344daf1f1e8c3fef886894482d2f).